shell/platform/darwin/ios/framework/Source/FlutterUmbrellaImport.m

```diff
@@ -2,6 +2,8 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
+// FLUTTER_NOLINT: https://github.com/flutter/flutter/issues/93360
```

Comment:
This one is new.
[`FlutterUmbrellaImport`](https://github.com/flutter/engine/blob/cf087516afb930de549e672b37a1e7d70b379931/shell/platform/darwin/ios/framework/Source/FlutterUmbrellaImport.m#L5-L9) is used in the [`copy_and_verify_framework_module` target](https://github.com/flutter/engine/blob/d57627c98fee5da1eb1a85ae29cbd083fbc98c5c/shell/platform/darwin/ios/BUILD.gn#L380), is not included in the engine framework source, and cannot be compiled on its own.

When `clang-tidy` is run on this file, it fails with a `clang-diagnostic-error` (compilation failure).  Add a `FLUTTER_NOLINT` to this file with link to https://github.com/flutter/flutter/issues/93360.